src/components/interactive-map/shapes/curved-arrow.tsx

```diff
@@ -109,7 +109,7 @@ const CurvedArrow: React.FC<CurvedArrowProps> = (props) => {
     y: end.y - props.arrowLength * Math.sin(endAngle),
   };
 
-  const centerPoint = getBezierPoint(0.5, start, cp1, cp2, end);
+  const centerPoint = getBezierPoint(0.3, start, cp1, cp2, end);
```

Comment:
Is this something like the "amount" of curve? I think we should pass it in as a prop.

Reply:
No, this is where the 'x' gets drawn on the support curve. 

If we do this at 0.5, this means that it usually overlaps with the actual move order. If we set it at about 0.3, it usually is where the support order does not yet overlap the actual move order. We could pass it as a prop, but I don't think we'll ever change this.